### ISSUE TITLE
Write a blog post about REST Data with Panache

### ISSUE THREAD

**Issue by gytis** [snowdrop/team#294]

**Comment by cmoulliard** [comment]
This task has been created in Nov 2020 and time has come to start to work it. WDYT ? @gytis

**Comment by gytis** [comment]
Yes, I was just thinking what to do with it. Whether to start it or to some how take it out of the report. 
It's good to have tickets so that they would remind of some work that needs to be done, but I don't like that they keep being included in the report.

**Comment by cmoulliard** [comment]
> but I don't like that they keep being included in the report.

Maybe move it then to the column `backlog` ?

**Comment by gytis** [comment]
It is in a backlog. The reports picks up any issue that is assigned. Doesn't matter in which column it is.

**Comment by cmoulliard** [comment]
> It is in a backlog. The reports picks up any issue that is assigned. Doesn't matter in which column it is.

Could we restrict the tasks that we collect for the report and only select "Weekly call" or "In Progress" or "Closed" AND not "Backlog" or "New" @jacobdotcosta ?

**Comment by jacobdotcosta** [comment]
ATM the bot doesn't integrate with zenhub and we'd have to take information from there. So we must investigate the integration with zenhub.

**Comment by jacobdotcosta** [comment]
Looking anxious to read a blog post about this subject...
